### ISSUE TITLE
too many buttons in RMarkdown tutorial toolbar

### ISSUE THREAD

**Issue by kevinushey** [rstudio/rstudio#3138]
### System details

    RStudio Edition : Desktop
    RStudio Version : v1.2.805
    OS Version      : macOS 10.13.5
    R Version       : 3.5.0

### Steps to reproduce the problem

Open an R Markdown document containing:

```r
---
title: "Tutorial"
output: 
  learnr::tutorial:
    progressive: true
    allow_skip: true
runtime: shiny_prerendered
---
```
### Describe the problem in detail

RStudio renders too many dropdown buttons (without accompanying text; also too squished)

Multiple screenshots just for context + contents of those dropdowns.

<img width="353" alt="screen shot 2018-07-10 at 1 03 28 pm" src="https://user-images.githubusercontent.com/1976582/42534587-e53fbdf4-8441-11e8-9b1c-7916b9414910.png">
<img width="367" alt="screen shot 2018-07-10 at 1 03 34 pm" src="https://user-images.githubusercontent.com/1976582/42534588-e5a639bc-8441-11e8-86d6-89851881965d.png">
<img width="356" alt="screen shot 2018-07-10 at 1 03 41 pm" src="https://user-images.githubusercontent.com/1976582/42534599-ebd4e108-8441-11e8-9187-2f7b1084fb97.png">

### Expected

Not exactly sure what the appropriate resolution here is; I suspect we don't want record + run tests for R Markdown tutorials?

**Comment by jmcphers** [comment]
Shiny R Markdown documents (`runtime: shiny`) can indeed be tested with shinytest. I'm less certain about prerendered shiny (`runtime: shiny_prerendered`). @wch will know for certain.

@javierluraschi should we just have everything in one dropdown here? (clear cache, record tests, etc.)

**Comment by dfalty** [comment]
Verified fixed on version 1.2.1090.
